Source/control.cpp

```diff
@@ -2881,21 +2881,24 @@ void control_reset_talk()
 
 int control_talk_last_key(int a1)

```

Comment:
Probably this should be returning BOOL
```suggestion
BOOL control_talk_last_key(int a1)
```
This is normally how bool will look for things that did not have a PSX match.